internal/backends/winit/glwindow.rs

```diff
@@ -662,6 +662,20 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
     fn dark_color_scheme(&self) -> bool {
         dark_light::detect() == dark_light::Mode::Dark
     }
+
+    fn is_visible(&self) -> bool {
+        if let Some(mapped_window) = self.borrow_mapped_window() {
+            match mapped_window
+                .canvas
+                .with_window_handle(&mut |win: &winit::window::Window| win.is_visible())
+            {
+                Some(visibility) => visibility,
+                None => false,
```

Comment:
maybe true is better here.  When the window is mapped, it is most likely visible.
(can be written `.unwrap_or(true)`)